x-pack/test/api_integration/deployment_agnostic/default_configs/stateful.config.base.ts

```diff
@@ -152,6 +152,13 @@ export function createStatefulTestConfig<T extends DeploymentAgnosticCommonServi
           '--xpack.uptime.service.username=localKibanaIntegrationTestsUser',
           '--xpack.uptime.service.devUrl=mockDevUrl',
           '--xpack.uptime.service.manifestUrl=mockDevUrl',
+          `--logging.loggers=${JSON.stringify([
+            {
+              name: 'plugins.observabilityAIAssistant',
+              level: 'all',
+              appenders: ['default'],
+            },
+          ])}`,
```

Comment:
is it a temporary change or we plan to merge with it? If that's a plan, could you share reason to have it for stateful?